.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+os:
+  - linux
+# ** patchelf isn't supported here, _yet_ **
+# addons:
+#   apt:
+#     sources:
+#       - ppa:utlemming/rax-playground
+#     packages:
+#       - build-essential
+#       - ninja-build
+#       - patchelf
+#       - libboost-dev
+#       - libboost-program-options-dev
+#       - libogg-dev
+before_install:
+    - sudo add-apt-repository ppa:utlemming/rax-playground -y
+    - sudo apt-get update -qq
+    - sudo apt-get install build-essential ninja-build libboost-dev libboost-program-options-dev libogg-dev patchelf
+install: pip install flake8 .
+script: python -m unittest discover -s test && python setup.py lint
```

Comment:
You should probably use `python setup.py test` here, since that's the primary interface to the tests, and it's always possible I'll do something different with the test framework (e.g. switch to nose). I'll always try to ensure that `python setup.py test` does the right thing, though.